server/Makefile

```diff
@@ -56,9 +63,8 @@ run-app:
 run-db:
 	docker compose -f ../docker-compose.yml up -d reearth-mongo
 
-gql:
-	go generate ./internal/adapter/gql/gqldataloader
-	go generate ./internal/adapter/gql
+generate:
+	go generate ./...
```

Comment:
[memo]
If we generate just only files related with gql, I concerned we can't notice whether other generator works correctly.
So I change this command to `generate` and `make generate` execute all `go generate`